src/func-aeh-listener/src/.editorconfig

```diff
@@ -10,6 +10,16 @@ indent_size = 4
 insert_final_newline = true 
 charset = utf-8-bom 
 
+############################### 
+# MS Coding Conventions       # 
+############################### 
+[*.{csproj,xml}]
+indent_style = space 
+end_of_line = lf
+indent_size = 2 
+insert_final_newline = true 
+charset = utf-8-bom 
```

Comment:
Do we have to use bom?

Reply:
its whats been used before, what would you suggest we change to?

Reply:
UTF 8 without the bom unless it's an MS prescribed thing, but minor, can discuss later if you'd prefer

Reply:
Updated 👍

Reply:
Cool nice one!